tests/test_generic/tests/test_industry_requirements.py

```diff
@@ -78,3 +79,19 @@ def test_sale_ok_and_is_published_in_db(self):
                         ["%s (ID: %s)" % (record.display_name, record.id) for record in records],
                     ),
                 )
+    def test_knowledge_article_links_use_latest(self):
+        pattern = r'https?://www\.odoo\.com/documentation/(?!latest/)(\d+\.\d+|master)/'
+        for module in self.installed_modules:
+            knowledge = self.env['ir.model.data'].search(
+                [('model', '=', 'knowledge.article'), ('module', '=', module)], limit=1
+            )
+            if not knowledge:
+                continue
+            knowledge_article = self.env.ref(knowledge.complete_name)
```

Comment:
@jaeschwa This will be replaced (when cloc is ready) by
```suggestion
            knowledge_article = self.env.ref('%s.welcome_article', raise_if_not_found=False)
            self.assertTrue(knowledge_article, "The knowledge article should have 'welcome_article' as id.")
```